packages/office-ui-fabric-react/src/components/Panel/examples/Panel.Scroll.Example.tsx

```diff
@@ -65,22 +63,22 @@ export class PanelScrollExample extends React.Component<
     );
   }
 
-  private _onClosePanel = () => {
-    this.setState({ showPanel: false });
-  };
-
   private _onRenderFooterContent = (): JSX.Element => {
     return (
       <div>
-        <PrimaryButton onClick={this._onClosePanel} style={{ marginRight: '8px' }}>
+        <PrimaryButton onClick={this._hidePanel} style={{ marginRight: '8px' }}>
           Save
         </PrimaryButton>
-        <DefaultButton onClick={this._onClosePanel}>Cancel</DefaultButton>
+        <DefaultButton onClick={this._showPanel}>Cancel</DefaultButton>
```

Comment:
Should this be `this._hidePanel`?

Reply:
Yep, I need to do a final round of testing and verification, great catch, I'll update this.

Reply:
Final round of testing and verification done!